Doc/README.rst

```diff
@@ -130,7 +130,7 @@ Contributing
 ============
 
 Bugs in the content should be reported to the
-`Python bug tracker <https://bugs.python.org>`_.
+`Python bug tracker <https://github.com/python/cpython/issues>`_.
```

Comment:
And in fact, shall we link to https://devguide.python.org/tracker/ instead? 

That has info about checking issues don't already exist, might help avoid duplicates.

```suggestion
`Python issue tracker <https://devguide.python.org/tracker/>`_.
```

Reply:
Much better!